docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-persistence-via-folder-action-script.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[prebuilt-rule-0-16-2-persistence-via-folder-action-script]]
+=== Persistence via Folder Action Script
+
+Detects modification of a Folder Action script. A Folder Action script is executed when the folder to which it is attached has items added or removed, or when its window is opened, closed, moved, or resized. Adversaries may abuse this feature to establish persistence by utilizing a malicious script.
```

Comment:
```suggestion
Detects the modification of a folder action script. A folder action script is executed when the folder it is attached to has items added or removed, or when its window is opened, closed, moved, or resized. Adversaries may abuse this feature to establish persistence by utilizing a malicious script.
```

Reply:
Removed some words in the second sentence that seemed out of place or accidentally left over. Ignore my edits if they weren't.